docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -91,71 +111,53 @@ The time required for the model to train is proportionate to the amount of data.
+If you're not satisfied with your accuracy metrics, some easy ways to try and improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, click **Next step** to navigate to the consume step.
 
-- TaxiFarePredictionML.ConsoleApp: A .NET Core Console application that contains the model training and sample consumption code.
-- TaxiFarePredictionML.Model: A .NET Standard class library containing the data models that define the schema of input and output model data, the saved version of the best performing model during training and a helper class called `ConsumeModel` to make predictions.
+## Consume the model
 
-1. In the code step of the Model Builder tool, select **Add Projects** to add the auto-generated projects to the solution.
-1. Open the *Program.cs* file in the *TaxiFarePrediction* project.
-1. Add the following using statement to reference the *TaxiFarePredictionML.Model* project:
+This step will have project templates that you can use to consume the model.
 
-    ```csharp
-    using System;
-    using TaxiFarePredictionML.Model;
-    ```
+- Console App
+- Web API
 
-1. To make a prediction on new data using the model, create a new instance of the `ModelInput` class inside the `Main` method of your application. Notice that the fare amount is not part of the input. This is because the model will generate the prediction for it.
-
-    ```csharp
-    // Create sample data
-    ModelInput input = new ModelInput()
-    {
-        Vendor_id = "CMT",
-        Rate_code = 1,
-        Passenger_count = 1,
-        Trip_distance = 3.8f,
-        Payment_type = "CRD"
-    };
-    ```
+### Console App
 
-1. Use the `Predict` method from the `ConsumeModel` class. The `Predict` method loads the trained model, create a [`PredictionEngine`](xref:Microsoft.ML.PredictionEngine%602) for the model and uses it to make predictions on new data.
-
-    ```csharp
-    // Make prediction
-    ModelOutput prediction = ConsumeModel.Predict(input);
-
-    // Print Prediction
-    Console.WriteLine($"Predicted Fare: {prediction.Score}");
-    Console.ReadKey();
-    ```
+When adding a console app to your solution, you will be prompted to name the project.
 
+1. Name the console project **TaxiFare_Console**.
+1. Click *Add to solution** to add the project to your current solution.
 1. Run the application.
 
     The output generated by the program should look similar to the snippet below:
 
     ```bash
-    Predicted Fare: 14.96086
+    Predicted Fare: 15.020833
     ```
 
-If you need to reference the generated projects at a later time inside of another solution, you can find them inside the `C:\Users\%USERNAME%\AppData\Local\Temp\MLVSTools` directory.
+### Web API
+
+When adding a web API to your solution, you will be prompted to name the project.
+
+1. Name the console project **TaxiFare_API**.
+1. Click *Add to solution** to add the project to your current solution.
+1. Run the application.
```

Comment:
Should we go over running something like Postman or Insomnia to test this or is that too much for this tutorial?

Reply:
Let's use PowerShell instead. Since MB is Windows only we can safely make the assumption everyone has PowerShell on their PC so they don't have to install anything else. 

https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/serve-model-web-api-ml-net#test-web-api-locally